checkov/cloudformation/cfn_utils.py

```diff
@@ -16,40 +17,44 @@
 CF_POSSIBLE_ENDINGS = frozenset([".yml", ".yaml", ".json", ".template"])
 
 
-def get_resource_tags(entity, registry=cfn_registry):
+def get_resource_tags(entity: Dict[str_node, dict_node], registry: Registry = cfn_registry) -> Optional[Dict[str, str]]:
     entity_details = registry.extract_entity_details(entity)
 
     if not entity_details:
         return None
 
     entity_config = entity_details[-1]
 
-    if type(entity_config) not in (dict, dict_node):
+    if not isinstance(entity_config, dict):
         return None
 
     try:
-        properties = entity_config.get('Properties')
+        properties = entity_config.get("Properties")
         if properties:
-            tags = properties.get('Tags')
+            tags = properties.get("Tags")
             if tags:
                 return parse_entity_tags(tags)
     except:
-        logging.warning(f'Failed to parse tags for entity {entity}')
+        logging.warning(f"Failed to parse tags for entity {entity}")
 
     return None
 
 
-def parse_entity_tags(tags):
-    if type(tags) == list_node:
-        tag_dict = {tag['Key']: str(get_entity_value_as_string(tag['Value'])) for tag in tags}
+def parse_entity_tags(tags: Union[list_node, Dict[str, Any]]) -> Optional[Dict[str, str]]:
+    if isinstance(tags, list_node):
+        tag_dict = {tag["Key"]: get_entity_value_as_string(tag["Value"]) for tag in tags}
         return tag_dict
     elif isinstance(tags, dict):
-        tag_dict = {str(key): str(get_entity_value_as_string(value)) for key, value in tags.items() if
-                    key not in ('__startline__', '__endline__')}
+        tag_dict = {
+            str(key): get_entity_value_as_string(value)
+            for key, value in tags.items()
+            if key not in ("__startline__", "__endline__")
```

Comment:
Please use the consts for this. I think we should move them from `checkov/cloudformation/context_parser.py` to here

Reply:
yeah, sure will do

Reply:
@nimrodkor I was not able to move them, because this would result in circular dependencies 🙁

Reply:
Another day then